2giwon/SearchMatrix.kt

```diff
@@ -0,0 +1,8 @@
+fun searchMatrix(matrix: Array<IntArray>, target: Int): Boolean {
+    if (matrix.isEmpty() || matrix[0].isEmpty()) return false
+    for (m in matrix) {
+        if (m.binarySearch(target, 0) >= 0) return true
```

Comment:
```suggestion
        if (m.binarySearch(target) >= 0) return true
```

두번째 파라미터는 생략해도 될것 같습니다 😄